angular-spa-ui/js/search-app/search-config.js

```diff
@@ -1,8 +1,22 @@
-"use strict";
-
-module.exports = function () {
-
-    var routCfg = {
-
+module.exports =
+    {
+        path: {
+            getUserPath : '/user/',
```

Comment:
Never name variables with "get..." or "set..." because these name-parts are only for functions and callbacks that applicable for getting or setting data.